### PR TITLE
Hotfix/release 0.4.0

### PR DESCRIPTION
**Pull request by beesaycheese**
Remove npm from the setup bash script and build the portal_build in the workflow during a pull request to the release and staging if there's any changes in the src/app folder

To view the results of the workflow, you can refer to this commit that was pushed by the workflow 
https://github.com/datature/portal/pull/64/commits/9a09a7ab47b9353d861caf44dbc69bf614915580

### CODE REVIEW

File: .github/workflows/build-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+      # Build NextJS Project
+      - name: Building Nexus / NextJS
+        working-directory: ./src/app
+        run: npm run build:static
+
+      - name: Commit files
+        shell: bash
+        run: |
+          git config --local user.email "beatrice.leong98@gmail.com"
```
Comment (beesaycheese) [inline .github/workflows/build-workflow.yml]: @gkeechin will have to change this. what should I change this to?

Comment (gkeechin) [reply .github/workflows/build-workflow.yml]: will this trigger an action recursion when the new commit triggers another cicd and so on?

Comment (gkeechin) [reply .github/workflows/build-workflow.yml]: Email -
```
41898282+github-actions[bot]@users.noreply.github.com
```
User Name
```
github-actions[bot]
```

File: .github/workflows/build-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Installing
+        working-directory: ./src/app
+        run: npm install
+
+      # Build NextJS Project
+      - name: Building Nexus / NextJS
+        working-directory: ./src/app
+        run: npm run build:static
+
+      - name: Commit files
```
Comment (gkeechin) [inline .github/workflows/build-workflow.yml]: `name : Pushing Static Files to Portal Build`

File: .github/workflows/build-workflow.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        shell: bash
+        run: |
+          git config --local user.email "beatrice.leong98@gmail.com"
+          git config --local user.name "beatrice"
+          git add -f ./portal_build/out
+          git commit -m "Add exported out dir" -a
```
Comment (gkeechin) [inline .github/workflows/build-workflow.yml]: `git commit -m "Re-Exported Portal to Portal Build`

**Review by gkeechin — CHANGES_REQUESTED** [review]

**Review by gkeechin — APPROVED** [review]

**Pull request merged**